index.js

```diff
@@ -5,5 +5,8 @@ module.exports = {
   PaymentMedium: require('./src/payment-medium'),
   Quote: require('./src/quote'),
   Trade: require('./src/trade'),
-  Helpers: require('./src/helpers')
+  Helpers: require('./src/helpers'),
+  mocks: {
```

Comment:
For use externally, maybe we could do something like this?
`let CoinifyTradeMock = require('bitcoin-coinify-client/mocks').Trade`
That way index.js doesn't contain references to any test-only code.